Xamarin.Forms.Core/WebView.cs

```diff
@@ -77,6 +79,12 @@ public CookieContainer Cookies
 			set { SetValue(CookiesProperty, value); }
 		}
 
+		public string setUserAgentString
```

Comment:
UserAgentString, property name should be a noun, when method should be a verb

Reply:
@VladislavAntonyuk I am using what was suggested in the Issue and the subsequent chat as the name, but feel free to suggest something better.
I must admit that I have only implemented the Setting of the UserAgentString hence the name making sense in that regard so possibly should remove the Getter.

Open to suggestions.